pkgs/tools/security/plasma-pass/default.nix

```diff
@@ -0,0 +1,39 @@
+{ mkDerivation, lib, fetchgit, cmake, extra-cmake-modules
+, ki18n
+, kitemmodels
+, oathToolkit
+, qgpgme
+, plasma-framework
+, qt5 }:
+
+mkDerivation rec {
+  pname = "plasma-pass";
+  version = "1.2.0";
+
+  src = fetchgit {
+    url = "https://invent.kde.org/plasma/plasma-pass";
+    rev = "v${version}";
+    sha256 = "1w2mzxyrh17x7da62b6sg1n85vnh1q77wlrfxwfb1pk77y59rlf1";
+  };
+
+  buildInputs  = [
+    ki18n
+    kitemmodels
+    oathToolkit
+    qgpgme
+    plasma-framework
+    qt5.qtbase
+    qt5.qtdeclarative
+  ];
+
+  nativeBuildInputs = [ cmake extra-cmake-modules ];
+
+  meta = with lib; {
+    description = "A Plasma applet to access passwords from pass, the standard UNIX password manager";
+    homepage = "https://github.com/KDE/plasma-pass";
+    license = licenses.lgpl21Plus;
+    maintainers = with maintainers; [ matthiasbeyer ];
```

Comment:
Please do not add other people.

Reply:
Nobody adds other people here.

Reply:
That's wrong. Someone can't just forward someones patch to nixpkgs from some other repo without consent of the author/maintainer. If you are opening this with your 2nd GitHub account please make this clear somewhere.

If this behavior would be allowed then packages could be associated to maintainers which never agreed with that and create a false hint to someone who is interested in the package.

You clearly have access to a GitHub account, so please use it to add packages you want to have in nixpkgs instead of proxying over 2 conners.

Reply:
> Someone can't just forward someones patch to nixpkgs from some other repo without consent of the author/maintainer

Nobody did. There's the `Signed-off-by` line in the patch. Case closed.

> You clearly have access to a GitHub account, so please use it to add packages/maintainerships you want to have in nixpkgs instead of proxying over 2 conners.

I submit patches the way I like, not the way you would like to command me to.

Reply:
> That's wrong. Someone can't just forward someones patch to nixpkgs from some other repo without consent of the author/maintainer.

I think you misspelled "free software" there. I suggest reading up on the [MIT license](https://github.com/NixOS/nixpkgs/blob/master/COPYING).

Reply:
> Nobody did. There's the Signed-off-by line in the patch. Case closed.

Anyone can add this. I can also submit a patch in your name with that line.

> I think you misspelled "free software" there.

Well, of course I could do that but we do not accept users adding other people as maintainers for obvious reasons.

Reply:
If @matthiasbeyer says he agrees, then it's ok. No matter how the patch was submitted.